### ISSUE TITLE
Exact Schelkunoff solid rod as the default coax inner conductor

### ISSUE THREAD

**Issue by gvcallen** [paramrf/paramrf#97]
## What to build

An exact Schelkunoff cylindrical-conductor formulation for coax, replacing Tesche as the default. Tesche stays available as the cheap, Bessel-free alternative and as an independent cross-check.

Schelkunoff eq. (65) for a solid rod, as a shape-factor entry in the conductor cross-section layer:

$$Z_s = \zeta_c\,\frac{I_0(\gamma a)}{I_1(\gamma a)},\qquad \gamma=\sqrt{j\omega\mu\sigma}$$

### Why this matters

Tesche is an equivalent-circuit interpolation, not the exact intermediate solution. Measured against exact Schelkunoff over 10-500 MHz, its attenuation error is:

| geometry | 10 MHz | 500 MHz | residual after removing a best-fit global scale |
|---|---|---|---|
| RG58 0.91/2.95 | -7.2% | -1.1% | **5.3%** |
| RG405 0.51/1.68 | -11.1% | -2.0% | **7.9%** |
| LMR400 2.74/7.24 | -2.5% | -0.3% | 1.9% |

The last column is the one that matters for fitting: this is a *shape* error and refitting conductivity does not absorb it. Worst-case error on a bare rod is ~20% in $\Re Z$ around $a/\delta\approx2$-5.

### Evaluating the Bessel ratio in JAX

JAX has no complex Bessel functions (`jax.scipy.special.i0e`/`i1e` reject complex dtypes), so this needs a hand-rolled evaluator. A prototype validated during research, which should be taken as the design:

```
if |x| < 20:   power series for I0 and I1, ~40 terms, float64
else:          1 + 1/(2x) + 3/(8x^2) + 3/(8x^3) + 63/(128x^4) + 27/(32x^5)
```

Measured against `scipy.special.ive` for complex $x$ at $\arg=45°$, $|x|\in[10^{-6},10^4]$: worst relative error **1.2e-7** at the switch point, <=1e-12 elsewhere. Through the full rod impedance from 1 kHz to 20 GHz: <=2e-8. The DC limit needs no special case — $I_0/I_1\to2/x$ gives $Z\to1/(\pi a^2\sigma)$ analytically. `jax.grad` is finite throughout; the derivative jump at the branch switch is 1.4e-4 relative, reducible by raising the threshold. Only $\gamma\to0$ needs the `jnp.where`-with-safe-argument pattern already used across the package.

## Acceptance criteria

- [ ] A Schelkunoff coaxial formulation exists and is the default; Tesche remains selectable
- [ ] The Bessel-ratio evaluator is validated against scipy across the full argument range, with the tolerance recorded per regime
- [ ] Attenuation matches scikit-rf `model='schelkunoff'` for representative cable geometries
- [ ] The DC limit reproduces $1/(\pi a^2\sigma)$ without a special case
- [ ] Gradients are finite w.r.t. conductivity and frequency, including at DC
- [ ] The Tesche docstring's claim that it reaches "the exact high-frequency skin-effect impedance" is corrected: it reaches $R_{dc}+Z_{hf}$ and never captures the $1/2\gamma a$ curvature term
- [ ] Docstrings cite Schelkunoff (1934) with the equation numbers used

## Blocked by

- #96
